### PR TITLE
CVE-2011-3376

### PR DESCRIPTION
**Pull request by dwelcheck**
Completed vulnerability history for Tomcat CVE-2011-3376.
There is no _dev_ branch for this repo yet, so I'm creating the pull request against master for now.

### CODE REVIEW

**Comment by andymeneely** [comment]
Added a dev branch just now and changed your pull request base myself. Thanks!!

**Comment by dwelcheck** [comment]
It appears the CI build is failing on a file I did not change -- CVE-2018-11784.yml.

**Comment by andymeneely** [comment]
Hi Devon - saw your email to Prof. Rabb and I figured we'd discuss it here. Looks like the results for the build on PR #25 is somehow coming in here? This seems like a Travis issue. The file isn't changed in my dev branch, and I can clearly see from the PR that it only impacts one file.

Never seen this before and we've done this assignment for 2 years now and hundreds of students. Weird.

Your build should be considered "passing" for grading reasons. If Stephen gets it wrong, refer him to this comment.

**Comment by dwelcheck** [comment]
Alright, thanks!  I have also never seen something like that happen.

File: cves/CVE-2011-3376.yml [hunk 1]
```diff
@@ -4,33 +4,33 @@ nickname_instructions: |
...
   alongside its CVE to make it more easily distinguished from the rest.
-nickname:
+nickname: Priviledge Escalation via Manager
 
 CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: 284
```
Comment (lukasyelle) [inline cves/CVE-2011-3376.yml:13]: Have 5 Upvotes! 👍🏻

Comment (dwelcheck) [reply cves/CVE-2011-3376.yml:13]: Done.

**Review by vasindaniil — COMMENTED** [review]
+2 upvotes

**Comment by dwelcheck** [comment]
> +2 upvotes
Done!

**Pull request merged**